mula/scheduler/server/server.py

```diff
@@ -470,6 +474,16 @@ def patch_task(self, task_id: str, item: Dict) -> Any:
                 detail="failed to update task",
             ) from exc
 
+        # Retrieve updated task
+        try:
+            updated_task = self.ctx.datastores.task_store.get_task_by_id(task_id)
+        except Exception as exc:
+            self.logger.error(exc)
+            raise fastapi.HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail="failed to get updated task",
+            ) from exc
+
```

Comment:
We want to make sure that when the runner updates the status of the task. That the duration in a `Task` is calculated and updated correctly